src/contract/userop/index.ts

```diff
@@ -0,0 +1,40 @@
+import { VerifiedWallet } from "../../wallet";
+import { USEROP_DATATYPES, UseropResponse, UseropVerifiedContract } from "./contract";
+
+interface TransactionParams{
+    value:string
+}
+interface ClassParams {
+    apiKey:string, 
+    paymasterUrl:string,
+    wallet:VerifiedWallet
+    signer:VerifiedWallet
+    // options?:Options
+}
+class GasLessTransaction extends UseropVerifiedContract {
+
+    constructor(params: ClassParams){
+        super(params.signer,params.apiKey,params.paymasterUrl,params.wallet)
+        
+        // Validate the address
+        this.validateInput(USEROP_DATATYPES.ADDRESS,params.wallet.address);
+
+        // Validate paymasterUrl
+        this.validateInput(USEROP_DATATYPES.STRING,params.paymasterUrl);
+
+        // Validate api key
+        this.validateInput(USEROP_DATATYPES.STRING,params.apiKey);
+
+    }
+
+    async makeTransaction(params:TransactionParams):Promise<UseropResponse>{
+         // Validate value input
+        this.validateInput(USEROP_DATATYPES.VALUE,params.value);
+
+       return await this.callContract(params.value)
+    }
```

Comment:
Based on my research, `new ethers.Contract(address, this.abiInterface, signer)` with `userop` is only required while making `batched calls` which require `tokenAdress` which can be got from `etherscan` but for  this SDK they are not required that's why I have avoided them

Reply:
Also `callContract` method doesn't receieve dynamic `function` because currently there is only one function to call but I can modify it accordingly in case we have another contract function to call.

Reply:
@herberthk 

new ethers.Contract() should not change number of parameters and how the constructor of the VerifiedContract currently works. Just modify callContract to use exported values of userops related configuration parameters such as paymaster and bundler urls, etc.